capstone-rs/src/instruction.rs

```diff
@@ -328,6 +345,64 @@ impl<'a> InsnDetail<'a> {
     }
 }
 
+impl<'a> InsnRegsAccess {
+    fn new(cs: csh, ins: &cs_insn) -> Self {
+        let mut regs_read = [0u16; 64];
```

Comment:
Use `cs_regs` type directly instead of manually declaring an array.

Reply:
Can you give an example how to initialize a variable with a type?

Reply:
It looks like `cs_regs` is a type alias:

~~~rust
pub type cs_regs = [u16; 64usize];
~~~

Just add an explicit type instead of using type inference:

~~~rust
let mut regs_read: cs_regs = [0u16; 64];
~~~